### ISSUE TITLE
DFL 2022 | Card listing (all cards) and aside

### ISSUE THREAD

**Issue by kevinhowbrook** [mozilla/foundation.mozilla.org#9394]
### Describe the feature request

We'll need a block to add the new cards from #9393. This will be a list block offering the choice of adding multiple cards and also optional "aside" details.

### Screenshots / mock-ups

![Screenshot(1)](https://user-images.githubusercontent.com/2128707/191257098-748f5de3-7617-48e9-892e-3365e824502e.png)

### Additional context

Rather than add the aside information at the page level - this is better added as an optional part of the block as it's tied to the design/layout.

Design similar to blog set: https://foundation.mofostaging.net/en/docs/how-do-i-wagtail/streamfield-blocks/#blog-set. 2 per row instead of 3

Link to [designs](https://www.figma.com/file/goD9bRqPoedQAxi8OsInkR/Data-Futures-Lab%3A-Designs?node-id=626%3A9963)

**Estimate**
4h

**Comment by kristinashu** [comment]
Is there a way to make the "aside" flexible enough to work next to other streamfield blocks as well, it seems limiting for it to only work with the listing card. Note: an "aside" was also design (not build yet) for the [research hub](https://www.figma.com/file/6BXnlCvPcQrtIaxunvuBYc/Mozilla-Research-Hub?node-id=1166%3A4959).

**Comment by tbrlpld** [comment]
> Is there a way to make the "aside" flexible enough to work next to other streamfield blocks as well, it seems limiting for it to only work with the listing card. Note: an "aside" was also design (not build yet) for the [research hub](https://www.figma.com/file/6BXnlCvPcQrtIaxunvuBYc/Mozilla-Research-Hub?node-id=1166%3A4959).


@kristinashu Was that link supposed to go to the detail page on the design? If so, the "project lead" and "collaborators" sections are not an aside element in this sense, it's just a location in the much more structural layout of that page. That element will always be at the same place (next to the overview). I am not sure if any of the implementation can be reused, since there is also no streamfield on that page.

**Comment by kristinashu** [comment]
Ok, thank you for clarifying Tibor!

**Comment by kevinhowbrook** [comment]
Have re-worked this to have a dedicated aside streamfield area https://github.com/mozilla/foundation.mozilla.org/pull/9434

**Comment by tbrlpld** [comment]
@kevinhowbrook I left some small questions and suggestions for the layout. But other than that it looks good 👍

**Comment by tbrlpld** [comment]
@cdanfon This is now on staging

**Comment by cdanfon** [comment]
Hey @kevinhowbrook tested on stage, a couple of things on this one, although looks like Simon Evans will be picking this on up:

1) The aside section doesn't quite level as per design (don't know if this is a dev constrain)
2) The aside button in the design is left aligned, whilst in the staging site is centred 


See attached 
<img width="1294" alt="Screenshot 2022-10-10 at 18 27 27" src="https://user-images.githubusercontent.com/102978346/194922347-417592bf-ccdc-4b49-9cb0-06efc34fe144.png">
<img width="823" alt="Screenshot 2022-10-10 at 18 29 24" src="https://user-images.githubusercontent.com/102978346/194922407-8d8f2629-3cc3-4090-afc0-210a657c36d9.png">

**Comment by kristinashu** [comment]
The card layout looks good!

**Comment by gutermul** [comment]
Tested. I could create the page, but don't feel like we can sign off until the Aside is finished. The only thing I noticed on functionality is that the link function only supports internal links (I needed to select an existing internal page, rather than just enter a URL). See Screenshots: 


![Image](https://user-images.githubusercontent.com/21171146/196937390-83415832-a67b-45c2-affe-0b764277dbcb.png)



![Image](https://user-images.githubusercontent.com/21171146/196937437-c20c1d09-2916-4d94-a018-f797a6632e41.png)

**Comment by cdanfon** [comment]
Hey @gutermul thanks for testing! 

If you tick the 'Aside' box underneath this section you can replicate how it looks in the Figma files:

![Screenshot 2022-10-20 at 13 49 44](https://user-images.githubusercontent.com/102978346/196953340-9e223dba-d3fb-44de-9b34-e46efd3506df.png)
![Screenshot 2022-10-20 at 13 48 12](https://user-images.githubusercontent.com/102978346/196953347-4b10142d-ecd0-4633-b7ea-ed8bd06f60bb.png)

Regarding the external link, this was not part of the ACs - we could create a new ticket for this and add it to the backlog for the future, but would be out of scope for this sprint. 

Are you happy to sign off on this based on the above screenshots? Thanks!

**Comment by gutermul** [comment]
I am waiting to see when the aside bar is finished to sign off. One thing I see right now is that the figma page looks like it's divided into thirds (vertically) and the layout on staging looks like the card listing takes up more than 2/3 of the page. That could be fine, but it's part of why I want to wait until the aside bar is finished.

**Comment by cdanfon** [comment]
Hey @gutermul the aside block is now on staging for you to test, so this should unblock this one :)
